samples/boot/oauth2/authcodegrant/src/main/resources/application.yml

```diff
@@ -0,0 +1,24 @@
+server:
+  port: 8080
+
+logging:
+  level:
+    root: INFO
+    org.springframework.web: INFO
+    org.springframework.security: INFO
+#    org.springframework.boot.autoconfigure: DEBUG
```

Comment:
Is this something that we keep in our samples to make it easy for folks to toggle it on and off while they are playing around with the sample?

Reply:
yes that's the motivation why I put it in there